examples/relay-chat/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    <title>JS-Waku WebRTC Relay</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="apple-touch-icon" href="./favicon.png" />
+    <link rel="manifest" href="./manifest.json" />
+    <link rel="icon" href="./favicon.ico" />
+  </head>
+
+  <body>
+    <div class="content">
+      <div class="header">
+        <h3>Status: <span id="status"></span></h3>
+
+        <h4><label for="remoteNode">Remote node multiaddr</label></h4>
+        <div>
+          <input
+            id="remoteNode"
+            value="/dns4/node-01.ac-cn-hongkong-c.go-waku.prod.statusim.net/tcp/443/wss/p2p/16Uiu2HAm1fVVprL9EaDpDw4frNX3CPfZu5cBqhtk9Ncm6WCvprpv"
```

Comment:
ah so actually we may not need to run our own go-waku node, right?

Reply:
yes, the main criteria is to have `circuit-relay` implemented on nodes and as of now it only works with `go-waku` nodes in prod fleet, the guide is to make sure folks know what to do in case it does not work due to low capacity or something